.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
should we rather test only on even versions? something that works on alternate version should most probably work on the skipped ones.

Reply:
I like to always test on oldest and latest at least. Even though you are correct, Python is known to introduce new features, deprecate others, and even break ABI in a micro release. (Ok, that last one happened only once but...)

TL;DR I'd wait to save CI time only/if we need in the future. For now, as long as the CIs are running OK, let's keep these. What do you think?